### ISSUE TITLE
[Issue]: rocPyDecode WHL is missing .so file

### ISSUE THREAD

**Issue by LakshmiKumar23** [ROCm/rocPyDecode#98]
### Problem Description

The rocPyDecode WHL is currently missing the `.so` file and only brings in the `.py` files. 
For clarification:
view of rocal whl file:
```
lakshmi@kapu:~/work/lk/rocAL/build$ python3
Python 3.10.12 (main, Sep 11 2024, 15:47:36) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pprint
>>> from zipfile import ZipFile
>>> path = 'amd_rocal-2.0.0-cp310-cp310-linux_x86_64.whl'
>>> names = ZipFile(path).namelist()
>>> pprint.pprint(names)
**['rocal_pybind.cpython-310-x86_64-linux-gnu.so',**
 'amd/__init__.py',
 'amd/rocal/__init__.py',
 'amd/rocal/decoders.py',
 'amd/rocal/fn.py',
 'amd/rocal/noise.py',
 'amd/rocal/pipeline.py',
 'amd/rocal/random.py',
 'amd/rocal/readers.py',
 'amd/rocal/reductions.py',
 'amd/rocal/types.py',
 'amd/rocal/plugin/__init__.py',
 'amd/rocal/plugin/generic.py',
 'amd/rocal/plugin/pytorch.py',
 'amd/rocal/plugin/tf.py',
 'amd_rocal-2.0.0.dist-info/METADATA',
 'amd_rocal-2.0.0.dist-info/WHEEL',
 'amd_rocal-2.0.0.dist-info/top_level.txt',
 'amd_rocal-2.0.0.dist-info/RECORD']
```

whereas in rocPyDecode is:
```
lakshmi@kapu:~/work/lk/rocPyDecode/dist$ python3
Python 3.10.12 (main, Sep 11 2024, 15:47:36) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pprint
>>> from zipfile import ZipFile
>>> path = 'rocPyDecode-1.0.0.60300-py3-none-manylinux_2_28_x86_64.whl'
>>> names = ZipFile(path).namelist()
>>> pprint.pprint(names)
['pyRocVideoDecode/__init__.py',
 'pyRocVideoDecode/__init__.pyi',
 'pyRocVideoDecode/decoder.py',
 'pyRocVideoDecode/demuxer.py',
 'pyRocVideoDecode/types.py',
 'rocPyDecode-1.0.0.60300.dist-info/LICENSE',
 'rocPyDecode-1.0.0.60300.dist-info/METADATA',
 'rocPyDecode-1.0.0.60300.dist-info/WHEEL',
 'rocPyDecode-1.0.0.60300.dist-info/top_level.txt',
 'rocPyDecode-1.0.0.60300.dist-info/RECORD']
```

setup is missing the `.so` from `package_data`
```
setup(
      name='rocPyDecode',
      description='AMD ROCm Video Decoder Library',
      url='https://github.com/ROCm/rocPyDecode',
      version='1.0.0' + '.' + get_rocm_rev(),
      author='AMD',
      license='MIT License',
      packages=['pyRocVideoDecode'],
      package_dir={'pyRocVideoDecode':'pyRocVideoDecode'},
      package_data={"pyRocVideoDecode":["__init__.pyi"]},
      cmdclass={'bdist_wheel': custom_bdist_wheel,},
      )
```

Needs to be updated to be similar to this
```
packages=find_packages(where='@TARGET_NAME@'),
    package_dir={'amd': '@TARGET_NAME@/amd'},
    include_package_data=True,
    ext_modules=[Extension('rocal_pybind',
                    sources=['rocal_pybind.cpp'], 
                    include_dirs=['@pybind11_INCLUDE_DIRS@', ROCM_PATH+'/include', '@PROJECT_SOURCE_DIR@/../rocAL/include/api'])],
```

### Operating System

Ubuntu

### CPU

All

### GPU

AMD Instinct MI300X

### ROCm Version

ROCm 6.2.0

**Comment by kiritigowda** [comment]
Fix in PR #99

**Comment by kiritigowda** [comment]
@LakshmiKumar23 will the whl generation fit into the existing pipeline for release with PR #99?

**Comment by essamROCm** [comment]
Fix is done in PR#99
